### DISCUSSION TITLE
Adicionar IP no BD do Azure

### DISCUSSION THREAD

**Discussion by JotaPeCarvalho** [balta-io/forum#245] [Tira Dúvidas] [answered]
Olá, não estou conseguindo achar o local para adicionar o IP da máquina no banco de dados SQL no azure, creio que desde que a aula foi gravada, ocorreu modificações, segue o print:
![image](https://user-images.githubusercontent.com/91575096/215531526-48384a33-1d58-4c4b-a4df-458a7ecf3d51.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @JotaPeCarvalho, como vai?

Acredito que você esteja no **Banco de Dados** e o IP é liberado no **Servidor de Banco de Dados**, na sessão **Network** ou **Rede**.
